src/views/Challenge.vue

```diff
@@ -88,8 +82,24 @@ export default {
         { k: { v: 'title', r: true}, l: this.$t('challenge.ZA') },
         { k: { v: no_ass, r: false}, l: this.$t('challenge.NO_REVIEWS_DESC') },
         { k: { v: no_ass, r: true}, l: this.$t('challenge.NO_REVIEWS_ASC') },
-        { k: { v: 'random', r: false}, l: this.$t('challenge.RANDOM') },
+        { k: { v: 'random', r: false}, l: this.$t('challenge.RANDOM') }
       ]
+      const walletsCondition = this.proposals.filter((prop) => prop.votes_cast).length > 0;
```

Comment:
The solution is correct and I like the fact that the existence of this additional fields is based on the data structure itself, it will be useful for future updates and it's always good to not hard-code conditions.
The only thing to consider is that filter in this way will loop every proposal in the challenge (and because it's in a computed property it can happen dinamically).
For this reason, assuming that the data structure is the same for each element, I would create new computed properties for these conditions checking only the first element of this.proposals. Something like:

```
withVoteCast(){
  if (this.proposals.length) {
    return this.proposals[0].votes_cast
  }
  return false
}
```
This will increase a bit the performance maintaining the same logic.